Godeps/_workspace/src/k8s.io/kubernetes/pkg/securitycontextconstraints/provider.go

```diff
@@ -228,20 +241,7 @@ func (s *simpleProvider) ValidateContainerSecurityContext(pod *api.Pod, containe
 		allErrs = append(allErrs, fielderrors.NewFieldInvalid("privileged", *sc.Privileged, "Privileged containers are not allowed"))
 	}
 
-	if sc.Capabilities != nil && len(sc.Capabilities.Add) > 0 {
-		for _, cap := range sc.Capabilities.Add {
-			found := false
-			for _, allowedCap := range s.scc.AllowedCapabilities {
-				if cap == allowedCap {
```

Comment:
I loosened it while testing to eliminate two things: ensuring the comparisons of allowed/required.add -> required.drop were flagging values despite case differences and to ensure that you wouldn't end up with duplicates in the same add/drop list.